dimod/bqm/utils.pyx

```diff
@@ -40,6 +42,78 @@ cdef object as_numpy_scalar(double a, np.dtype dtype):
     """Note that the memory is interpreted to match dtype, not a cast"""
     return PyArray_Scalar(&a, dtype, None)
 
+def cylmin(cyBQM bqm, **kwargs):
```

Comment:
Probably better to name the keyword argument explicitly, e.g `cylmin(cyBQM bqm, object default=None)`. The `object` type specifier is a cython thing, but better explicit than implicit...

One reason to do it this way it that it helps catch hard-to-find errors like
```
cylmin(bqm, dfault=1)
```

Reply:
It would also let people use positional arguments, e.g.
```
cylmin(bqm, 10)
```

Reply:
For posterity, the only way to handle the (edge) case of `None` (or any other sentinel) as the actual default value is not to depend on argument value, but existence of argument; therefore the need for `**kwargs`.

But since this is such a rare edge case, I'd agree explicitness trumps it here.

Reply:
FWIW, both unknown kwarg case and `default` as positional argument can be handled "manually."